addons/web_tour/static/src/tour_service/tour_helpers.js

```diff
@@ -114,7 +114,7 @@ export class TourHelpers {
         await dragEffectDelay();
         const target = await hoot.waitFor(selector, {
             visible: true,
-            timeout: 500,
+            timeout: 1000,
```

Comment:
I'm guessing this is because sometimes the target is animated or something like that.
The timeout is fine, but the PR will be better without the change